gittrigger/gittrigger.go

```diff
@@ -35,12 +35,36 @@ func (g *gitTrigger) getLatestCommit() error {
 	return nil
 }
 
-func (g gitTrigger) checkTrigger() bool {
+func (g *gitTrigger) GetRemote() (string, error) {
+	var out bytes.Buffer
+	cmd := exec.Command("bash", "-c", "git remote | head -n 1")
```

Comment:
This probably isn't perfect, but good enough. Depending on the alphabetical ordering the first remote may not be the real remote. Terrible example to demonstrate:
```
$ git clone https://github.com/rainforestapp/rainforest-cli.git .
$ git remote add aaa http://example.org
$ git remote
aaa
origin
```

You'd need to do something like this SO question to be truly correct: https://stackoverflow.com/questions/12609415/find-out-which-remote-a-local-branch-is-tracking, but it gets tricky because it seems some of the commands are only supported with certain versions of git (though the question is old-ish so maybe we can just assume it'd always work)


Also, I don't think this needs to assume bash? Or avoid shell shenanigans entirely and get the last line using go directly (like you do further down to get the first line from the `git remote get-url` command).
```suggestion
	cmd := exec.Command("sh", "-c", "git remote | head -n 1")
```

Reply:
Yeah, bash is very possibly not available in containers + new OSX releases. `|` will fail under `command.com` - although that's prolly niche for us.